### PR TITLE
Implement more PCC tests #205

### PR DESCRIPTION
**Pull request by Serranya**
This PR adds the remaining test classes to the PCC test cases. #205

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#304](https://github.com/yegor256/jpeek/issues/304) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#304](https://github.com/yegor256/jpeek/issues/304) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/51702245-aaa8-4f50-8369-f0d512454d48); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/jpeek/MetricsTest.java [hunk 1]
```diff
@@ -146,7 +145,29 @@
             new Object[] {"OverloadMethods", "LCOM3", 0.25d},
             new Object[] {"TwoCommonAttributes", "LCOM3", 1.0d},
             new Object[] {"WithoutAttributes", "LCOM3", 0.0d},
+            new Object[] {"Bar", "PCC", 0.5d},
```
Comment (fabriciofx) [inline src/test/java/org/jpeek/MetricsTest.java]: @Serranya By default in Java, floating point literals are double. So, the `d` in the literal is redundant. You can check it [here](https://stackoverflow.com/questions/41289696/floating-point-literal-floating-literal-double-literal/41289715).

File: src/test/java/org/jpeek/MetricsTest.java [hunk 2]
```diff
@@ -146,7 +145,29 @@
...
+            new Object[] {"BridgeMethod", "PCC", Double.NaN},
+            new Object[] {
+                "ClassWithDifferentMethodVisibilities", "PCC", Double.NaN,
+            },
+            new Object[] {"ClassWithPublicField", "PCC", Double.NaN},
+            new Object[] {"Foo", "PCC", 0.5d},
```
Comment (fabriciofx) [inline src/test/java/org/jpeek/MetricsTest.java]: @Serranya Same above.

File: src/test/java/org/jpeek/MetricsTest.java [hunk 3]
```diff
@@ -146,7 +145,29 @@
...
+            new Object[] {"IndirectlyRelatedPairs", "PCC", Double.NaN},
+            new Object[] {"MethodMethodCalls", "PCC", 0.2d},
```
Comment (fabriciofx) [inline src/test/java/org/jpeek/MetricsTest.java]: @Serranya Same above.

File: src/test/java/org/jpeek/MetricsTest.java [hunk 4]
```diff
@@ -146,7 +145,29 @@
...
             new Object[] {"MethodsWithDiffParamTypes", "PCC", 0.3333d},
+            new Object[] {"NoMethods", "PCC", Double.NaN},
+            new Object[] {"NotCommonAttributes", "PCC", Double.NaN},
+            new Object[] {
+                "NotCommonAttributesWithAllArgsConstructor", "PCC", 1.0d,
```
Comment (fabriciofx) [inline src/test/java/org/jpeek/MetricsTest.java]: @Serranya Same above.

File: src/test/java/org/jpeek/MetricsTest.java [hunk 5]
```diff
@@ -146,7 +145,29 @@
...
+            },
+            new Object[] {"OneCommonAttribute", "PCC", 0.5d},
```
Comment (fabriciofx) [inline src/test/java/org/jpeek/MetricsTest.java]: @Serranya Same above.

File: src/test/java/org/jpeek/MetricsTest.java [hunk 6]
```diff
@@ -146,7 +145,29 @@
...
+            new Object[] {"OneMethodCreatesLambda", "PCC", Double.NaN},
+            new Object[] {"OneVoidMethodWithoutParams", "PCC", Double.NaN},
+            new Object[] {"OnlyOneMethodWithParams", "PCC", 1.0d},
```
Comment (fabriciofx) [inline src/test/java/org/jpeek/MetricsTest.java]: @Serranya Same above.

File: src/test/java/org/jpeek/MetricsTest.java [hunk 7]
```diff
@@ -146,7 +145,29 @@
...
+            new Object[] {"OverloadMethods", "PCC", 3.0d},
```
Comment (fabriciofx) [inline src/test/java/org/jpeek/MetricsTest.java]: @Serranya Same above.

File: src/test/java/org/jpeek/MetricsTest.java (hunk 1 above)
Comment (Serranya) [reply src/test/java/org/jpeek/MetricsTest.java]: @fabriciofx I wanted to stay consistent with the rest of the formatting. I think we should either keep the `d` suffix in the added test cases, or also remove the `d` in all the other test cases in this file.

wdyt?

**Comment by Serranya** [comment]
@paulodamaso please take a look at this PR.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
@cucumber007[/z](https://www.0crat.com/u/cucumber007) please review this job completed by @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#304](https://github.com/yegor256/jpeek/issues/304) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by paulodamaso** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Pull request merged**